docs/usage.rst

```diff
@@ -302,6 +303,103 @@ Alice is the former owner:
         transfer_asset_id = transfer_tx['asset']['id']
 
 
+
+Recap: Asset Creation & Transfer
+--------------------------------
+
+.. code-block:: python
+
+	from bigchaindb_driver import BigchainDB
+	from bigchaindb_driver.crypto import generate_keypair
+	from time import sleep
+	from sys import exit
+	
+	alice, bob = generate_keypair(), generate_keypair()
+	
+	bdb = BigchainDB('http://localhost:59984/api/v1')
+	
+	bicycle_asset = {
+		'data': {
+			'bicycle': {
+				'serial_number': 'abcd1234',
+				'manufacturer': 'bkfab'
+			},
+		},
+	}
+	
+	bicycle_asset_metadata = {
+		'planet': 'earth'
+	}
+	
+	prepared_creation_tx = bdb.transactions.prepare(
+		operation='CREATE',
+		signers=alice.public_key,
+		asset=bicycle_asset,
+		metadata=bicycle_asset_metadata
+	)
+	
+	fulfilled_creation_tx = bdb.transactions.fulfill(
+		prepared_creation_tx,
+		private_keys=alice.private_key
+	)
+	
+	sent_creation_tx = bdb.transactions.send(fulfilled_creation_tx)
+	
+	txid = fulfilled_creation_tx['id']
+	
+	trials = 0
+	while trials < 60:
+		try:
+			if bdb.transactions.status(txid).get('status') == 'valid':
+				print('Tx valid in:', trials, 'secs')
+				break
+		except bigchaindb_driver.exceptions.NotFoundError:
+			trials += 1
+			sleep(1)
+	
+	if trials == 60:
+		print('Tx is still being processed... Bye!')
+		exit(0)
+	
+	asset_id = txid
+	
+	transfer_asset = {
+		'id': asset_id
+	}
+	
+	output_index = 0
+	output = fulfilled_creation_tx['outputs'][output_index]
+	
+	transfer_input = {
+		'fulfillment': output['condition']['details'],
+		'fulfills': {
+			'output': output_index,
+			'txid': fulfilled_creation_tx['id']
+		},
+		'owners_before': output['public_keys']
+	}
+	
+	prepared_transfer_tx = bdb.transactions.prepare(
+		operation='TRANSFER',
+		asset=transfer_asset,
+		inputs=transfer_input,
+		recipients=bob.public_key,
+	)
+	
+	fulfilled_transfer_tx = bdb.transactions.fulfill(
+		prepared_transfer_tx,
+		private_keys=alice.private_key,
+	)
+	
+	sent_transfer_tx = bdb.transactions.send(fulfilled_transfer_tx)
+	
+	print("Is Bob the owner?",
+		sent_transfer_tx['outputs'][0]['public_keys'][0] == bob.public_key)
+	
+	print("Was Alice the previous owner?",
+		fulfilled_transfer_tx['inputs'][0]['owners_before'][0] == alice.public_key)
+
```

Comment:
I think it might be nice to add a few comments in this block, to denote separate parts that people might be interested in.

For example, we could denote that we're going to make a `CREATE` transaction, then note that we'll be sending it (`bdb.transactions.send`), and then check its status (`bdb.transactions.status`), etc.